src/Console/Command/ListFixersCommand.php

```diff
@@ -0,0 +1,551 @@
+    private bool $hideInherited;
+    private bool $hideDeprecated;
+    private bool $hideCustom;
+    private int $countConfiguredFixers = 0;
+    private int $countRiskyFixers = 0;
+    private int $countEnabledFixers = 0;
+    private int $countInheritedFixers = 0;
+    private int $countDeprecatedFixers = 0;
+    private int $countCustomFixers = 0;
+
+    public function __construct(ToolInfoInterface $toolInfo, FixerFactory $fixerFactory = null)
+    {
+        parent::__construct(self::NAME);
+
+        $this->toolInfo = $toolInfo;
+
+        if (null === $fixerFactory) {
+            $fixerFactory = new FixerFactory();
+            $fixerFactory->registerBuiltInFixers();
+        }
+
+        $this->fixerFactory = $fixerFactory;
+
+        $this->fixerNameValidator = new FixerNameValidator();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure(): void
+    {
+        $optConfig = self::OPT_CONFIG;
+        $optOnlyConfigured = self::OPT_ONLY_CONFIGURED;
+        $optHideConfigured = self::OPT_HIDE_CONFIGURED;
+        $optHideEnabled = self::OPT_HIDE_ENABLED;
+        $optHideRisky = self::OPT_HIDE_RISKY;
+        $optHideInherited = self::OPT_HIDE_INHERITED;
+        $optHideDeprecated = self::OPT_HIDE_DEPRECATED;
+        $optHideCustom = self::OPT_HIDE_CUSTOM;
+        $optHideInheritance = self::OPT_HIDE_INHERITANCE;
+        $optCompare = self::OPT_COMPARE;
+
+        $this
+            ->setDefinition(
+                [
+                    new InputOption(self::OPT_CONFIG, '', InputOption::VALUE_REQUIRED, 'The path to a config file.'),
+                    new InputOption(self::OPT_ONLY_CONFIGURED, '', InputOption::VALUE_NONE, 'Show only fixers configured explicitly'),
+                    new InputOption(self::OPT_HIDE_CONFIGURED, '', InputOption::VALUE_NONE, 'Hide fixers that are configured (in the config file or because of inheritance).'),
+                    new InputOption(self::OPT_HIDE_ENABLED, '', InputOption::VALUE_NONE, 'Hide fixers that are currently enabled (the ones that are not disabled with [\'fixer_name\' => false]).'),
+                    new InputOption(self::OPT_HIDE_RISKY, '', InputOption::VALUE_NONE, 'Hide fixers that are marked as risky.'),
+                    new InputOption(self::OPT_HIDE_INHERITED, '', InputOption::VALUE_NONE, 'Hide fixers that inherited from RuleSets.'),
+                    new InputOption(self::OPT_HIDE_DEPRECATED, '', InputOption::VALUE_NONE, 'Hide fixers that are deprecated.'),
+                    new InputOption(self::OPT_HIDE_CUSTOM, '', InputOption::VALUE_NONE, 'Hide fixers that are custom.'),
+                    new InputOption(self::OPT_HIDE_INHERITANCE, '', InputOption::VALUE_NONE, 'Hide the addition inheritance information.'),
+                    new InputOption(self::OPT_COMPARE, '', InputOption::VALUE_NONE, 'Dumps the comparing result between your config and all available fixers in a copy-and-pastable format ready for the .php_cs file.'),
```

Comment:
If we really need to use variables to put them in the help section, then let's re-use assignment in options' definitions. I believe @keradus doesn't like this approach, but IMHO it's better than several additional lines of code.

```suggestion
        $this
            ->setDefinition(
                [
                    new InputOption($optConfig = self::OPT_CONFIG, '', InputOption::VALUE_REQUIRED, 'The path to a config file.'),
                    new InputOption($optOnlyConfigured = self::OPT_ONLY_CONFIGURED, '', InputOption::VALUE_NONE, 'Show only fixers configured explicitly'),
                    new InputOption($optHideConfigured = self::OPT_HIDE_CONFIGURED, '', InputOption::VALUE_NONE, 'Hide fixers that are configured (in the config file or because of inheritance).'),
                    new InputOption($optHideEnabled = self::OPT_HIDE_ENABLED, '', InputOption::VALUE_NONE, 'Hide fixers that are currently enabled (the ones that are not disabled with [\'fixer_name\' => false]).'),
                    new InputOption($optHideRisky = self::OPT_HIDE_RISKY, '', InputOption::VALUE_NONE, 'Hide fixers that are marked as risky.'),
                    new InputOption($optHideInherited = self::OPT_HIDE_INHERITED, '', InputOption::VALUE_NONE, 'Hide fixers that inherited from RuleSets.'),
                    new InputOption($optHideDeprecated = self::OPT_HIDE_DEPRECATED, '', InputOption::VALUE_NONE, 'Hide fixers that are deprecated.'),
                    new InputOption($optHideCustom = self::OPT_HIDE_CUSTOM, '', InputOption::VALUE_NONE, 'Hide fixers that are custom.'),
                    new InputOption($optHideInheritance = self::OPT_HIDE_INHERITANCE, '', InputOption::VALUE_NONE, 'Hide the addition inheritance information.'),
                    new InputOption($optCompare = self::OPT_COMPARE, '', InputOption::VALUE_NONE, 'Dumps the comparing result between your config and all available fixers in a copy-and-pastable format ready for the .php_cs file.'),
```

Alternatively you could use `sprintf()` to generate help section, but if you do this, then please use ordered placeholders (like `%1$s`) to not pass same value several times.

Reply:
@Wirone , no, it is not mandatory to use variables/constants: it is only something that is comfortable in my opinion, but not mandatory.

Can we ask the @keradus ' opinion before continuing?